src/cpp/session/modules/NotebookData.R

```diff
@@ -292,24 +292,36 @@
         paste0("<", summary, ">")
       })
 
+  # R 3.6.0 on Windows has an issue where RGui escapes can 'leak'
+  # into encoded strings; detect and remove those post-hoc
+  needsEncodeFix <-
+    .Platform$OS.type == "windows" &&
+    getRversion() == "3.6.0"
```

Comment:
Is the issue fixed in R 3.6.1?

Reply:
It is -- https://github.com/wch/r-source/commit/540cb48edea06ea710549cc98fa9966946a59c3f. I'll add that to the code to make it clear.